dhctl/pkg/kubernetes/actions/converge/static_node_group_controller.go

```diff
@@ -0,0 +1,130 @@
+// Copyright 2024 Flant JSC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package converge
+
+import (
+	"fmt"
+
+	"github.com/deckhouse/deckhouse/dhctl/pkg/config"
+	"github.com/deckhouse/deckhouse/dhctl/pkg/log"
+	"github.com/deckhouse/deckhouse/dhctl/pkg/terraform"
+	"github.com/deckhouse/deckhouse/dhctl/pkg/util/tomb"
+)
+
+type StaticNodeGroupController struct {
+	*NodeGroupController
+}
+
+func NewStaticNodeGroupController(controller *NodeGroupController) *StaticNodeGroupController {
```

Comment:
We have static nodegroup and i was confused with this name. 
I think better name is `NotMasterNodeGroupController(`